doc/migrations/3.1.1_3.1.1++.md

```diff
@@ -10,3 +10,26 @@
 ## **`ejabberd.cfg` renamed to `mongooseim.cfg`**
 
 Rename the existing config file of MongooseIM from `ejabberd.cfg` to `mongooseim.cfg`.
+
+## Pools configuration
+
+The way how pools to external service are configured changed, please see [Outgoing Connection doc](../advanced-configuration/outgoing-connections.md) for more details.
```

Comment:
Configuring pools to external services has changed, please see [Outgoing Connection doc](../advanced-configuration/outgoing-connections.md) for more details.